zazu/repo/commands.py

```diff
@@ -51,10 +55,22 @@ def ci(ctx):
 
 
 @repo.command()
+@click.argument('repository')
+@click.option('--nohooks', is_flag=True)
 @click.pass_context
-def clone(ctx):
+def clone(ctx, repository, nohooks):
     """Clone and initialize a repo"""
-    raise NotImplementedError
+    try:
+        Git().clone(repository)
+        click.echo('Repository successfully cloned')
+
+        if not nohooks:
+            click.echo('Installing Git Hooks')
+            zazu.git_helper.install_git_hooks(repository.rsplit('/', 1)[-1].replace('.git', ''))
```

Comment:
I believe you can get the handle to the repo from the clone call above, then use that to get the path to the repo. Seems more robust that way.